### PR TITLE
Changing the UI of height converter

### PR DESCRIPTION
**Pull request by Drishti-jain21**
### 👨‍💻 Changes proposed
I changed the background image, improved the UI of form and added text message when height is converted
It even has a reset button, which was not there earlier.
image

### ✅ Check List (Check all the applicable boxes) 

- [x] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [x] This PR does not contain plagiarized content.
- [x] My Addition/Changes works properly and matches the overall repo pattern.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by myhacktech — APPROVED** [review]
@Astrodevil

**Comment by Astrodevil** [comment]
Not working properly

**Comment by Drishti-jain21** [comment]
It is working fine on my system, I just checked

**Comment by Drishti-jain21** [comment]
image

image

image

File: Public/height-converter.html [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
-
-
-  </div>
 
-
-
-
-
-<script src ="../assets/Js/height-converter.js"> </script>
+<body style="background-image: url('/assets/Images/heightconverter.jpg');">
```
Comment (Anmol-Baranwal) [inline Public/height-converter.html:15]:
```suggestion
<body style="background-image: url('../assets/Images/heightconverter.jpg');">
```

File: Public/height-converter.html [hunk 2]
```diff
@@ -1,37 +1,45 @@
 <!DOCTYPE html>
 <html>
 <head>
-   <title>Height Converter</title>
-   <link rel="stylesheet" type="text/css" href="../assets/css/height-converter.css">
-
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="/assets/css/height-converter.css">
```
Comment (Anmol-Baranwal) [inline Public/height-converter.html:7]:
```suggestion
    <link rel="stylesheet" href="../assets/css/height-converter.css">
```

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
Please make these changes, then it would work properly.

**Comment by Anmol-Baranwal** [comment]
@Astrodevil 

I prefer the previous UI as it was simple and present image has no appropriate styling and doesn't really signifies anything. 
Still, I would merge the PR if the above changes are done.

**Pull request closed without merging**